patch/0012-arm64-dts-marvell-Add-Supermicro-SSE-G3748-board.patch

```diff
@@ -0,0 +1,145 @@
+From a2c35e835144054d553469f055522b49dae03bd3 Mon Sep 17 00:00:00 2001
+From: SelvaSupermicro  <selvarajr@supermicro.com>
```

Comment:
Please add your real name and use one space.

    $ git config --global user.name "Selva Ra…"
    $ git commit --amend -s --author="Selva … <selvarajr@supermicro.com>"